cmd/krew/cmd/search.go

```diff
@@ -30,6 +30,11 @@ import (
 	"sigs.k8s.io/krew/internal/installation"
 )
 
+type searchTarget struct {
+	names        []string
+	descriptions []string
+}
```

Comment:
I meant 

```
type searchTarget struct {
	name        string
	description string
}
```

does that not make more sense?

Reply:
Although I see we pass `fuzzy.Find(keyword, descriptions)`, in that case, maybe the best course of action is to specify a type like:

```
type searchItem struct { name, description string }
type searchCorpus []searchItem
func (searchCorpus) descriptions() []string
func (searchCorpus) names() []string
```

https://pkg.go.dev/golang.org/x/exp/slices and https://pkg.go.dev/golang.org/x/exp/maps (you can bring them) may have some methods to deal with

Reply:
Updated.